randovania/games/dread/json_data/Cataris.txt

```diff
@@ -4615,7 +4628,7 @@ Extra - asset_id: collision_camera_055
 > Main Room; Heals? False
   * Layers: default
   > Tile Group (MISSILE)
-      Grapple Beam or Spider Magnet or Use Spin Boost
+      Grapple Beam or Spider Magnet or Speed Booster or Use Spin Boost
```

Comment:
Grapple Beam should have Grapple Movement with it

Reply:
you can grapple right next to the wall and just let go and grab the ledge, shooting the missiles beforehand, do we really need extra stuff for that?